.github/ISSUE_TEMPLATE/hass_discovery.md

```diff
@@ -1,29 +1,29 @@
 ---
-name: Home Assistant discovery issue
-about: Zwave-js integration and MQTT discovery issues
-title: "[HASS]"
+name: Home Assistant Discovery Issues
+about: Z-Wave JS integration and MQTT discovery issuess
+title: "[Home Assistant]"
 labels: discovery
 assignees: billiaz
 
```

Comment:
dito, let's turn this into an issue form

Reply:
Agree. https://github.com/zwave-js/node-zwave-js/blob/master/.github/ISSUE_TEMPLATE/bug_report.yml

Reply:
Same for this